tgrun/pkg/runner/vmi/embed/testhelpers.sh

```diff
@@ -90,6 +91,12 @@ function dump_longhorn_logs() {
     done
 }
 
+function rook_ceph_object_store_info() {
+    export OBJECT_STORE_ACCESS_KEY=$(kubectl -n rook-ceph get secret rook-ceph-object-user-rook-ceph-store-kurl -o yaml | grep AccessKey | head -1 | awk '{print $2}' | base64 --decode)
+    export OBJECT_STORE_SECRET_KEY=$(kubectl -n rook-ceph get secret rook-ceph-object-user-rook-ceph-store-kurl -o yaml | grep SecretKey | head -1 | awk '{print $2}' | base64 --decode)
+    export OBJECT_STORE_CLUSTER_IP=$(kubectl -n rook-ceph get service rook-ceph-rgw-rook-ceph-store | tail -n1 | awk '{ print $3}')
+}
```

Comment:
Could we make `rook_ecph_object_store_info` call this one instead of duplicating the code?